tools/strip_nibabel

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Convert NiBabel objects in PyMVPA datasets attributes into a portable form
+
+PyMVPA datasets that contain NiBabel objects, such as image header instances,
+as it was default prior PyMVPA 2.4, can suffer from NiBabel API changes that
+result in the inability to re-load serialized datasets (e.g. from HDF5) with
+newer versions of NiBabel.
+
+This script converts these NiBabel internals into a simpler form that helps
+process such datasets with a much wider range of NiBabel Versions, and
+removes the need to have NiBabel installed for simply loading such a dataset.
+
+If you run into this problem, you need access to NiBabel source code that
+can still process your dataset. A reasonable guess is to try NiBabel
+version 1.1. Here is a recipe:
+
+Get the NiBabel repository from github and checkout the version you need::
+
+  % git clone  https://github.com/nipy/nibabel /tmp/nibabel
+  % cd /tmp/nibabel
+  % git checkout 1.1.0
+
+Now call this script with the path to your old NiBabel, the HDF5 file with
+the to-be-converted datasets, and the output filename.
+
+  % tools/strip_nibabel /tmp/nibabel olddata.hdf5 newdata.hdf5
```

Comment:
Seems overkill for a one time effort. I have no experience with virtualenv, but feel free to tweak. From my POV I have no preference for either git or virtualenv -- unless virtualenv would pull in tons of 3rd-party packages before it could do the job. The currently implemented approach takes seconds to be functional.